lib/class-wp-rest-widget-updater-controller.php

```diff
@@ -180,6 +236,46 @@ public function compute_new_widget( $request ) {
 			)
 		);
 	}
+
+	/**
+	 * Returns the new widget instance and the form that represents it.
+	 *
+	 * @since 5.2.0
+	 * @access public
+	 *
+	 * @param WP_REST_Request $request Full details about the request.
+	 * @return WP_REST_Response|WP_Error Response object on success, or WP_Error object on failure.
+	 */
+	public function compute_new_widget( $request ) {
+		$identifier         = $request->get_param( 'identifier' );
+		$is_callback_widget = $request->get_param( 'is_callback_widget' );
```

Comment:
I believe this should be included in `args` for the `register_routes` function?

Reply:
I was not sure if args should only be used for arguments part of the route itself e.g: /(?P<identifier>[\w-_]+)/. But in my tests, it seems args also work on arguments part of the post data, so I updated the code to follow your suggestion.